lib/telephony/pinpoint/sms_sender.rb

```diff
@@ -94,7 +94,7 @@ def phone_info(phone_number)
           )
           break if response
         rescue Seahorse::Client::NetworkingError,
-               Aws::Pinpoint::Errors::InternalServerErrorException => error
+               Aws::Pinpoint::Errors::ServiceError => error
```

Comment:
related, we do this correctly elsewhere: https://github.com/18F/identity-idp/blob/1942c25a3137827c84a4fe50dd0f1839748ffc1a/lib/telephony/pinpoint/sms_sender.rb#L62